### PR TITLE
[12.0][ADD] product_intercompany_account

### PR DESCRIPTION
**Pull request by MateuGForgeFlow**
Change the income and expense accounts when dealing with intercompany transactions.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
@MateuGForgeFlow can you attend to Travis, and add tests?

**Comment by MateuGForgeFlow** [comment]
@JordiBForgeFlow On it!

File: intercompany_accounts/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    property_account_income_intercompany = fields.Many2one(
+        comodel_name="account.account",
+        company_dependent=True,
+        string="Income Intercompany Account",
+        help="Keep this field empty to use the default value from the product category."
```
Comment (LoisRForgeFlow) [inline intercompany_accounts/models/product_template.py]: please update this help message.

File: intercompany_accounts/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    )
+    property_account_expense_intercompany = fields.Many2one(
+        comodel_name="account.account",
+        company_dependent=True,
+        string="Expense Intercompany Account",
+        help="Keep this field empty to use the default value from the product category."
```
Comment (LoisRForgeFlow) [inline intercompany_accounts/models/product_template.py]: please update this help message.

**Review by LoisRForgeFlow — CHANGES_REQUESTED** [review]
Code review looks good. However the name of the module should not contain plurals and it can be more descriptive IMO because `intercompany_account` would be too generic. My suggestions: `account_invoice_intercompany_account` or `product_intercompany_account`.

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM!

**Review by JoanSForgeFlow — APPROVED** [review]
LGTM! :+1:

**Review by AdriaGForgeFlow — APPROVED** [review]

**Pull request merged**